lib/browser/api/touch-bar.js

```diff
@@ -180,6 +180,10 @@ TouchBar.TouchBarPopover = class TouchBarPopover extends TouchBarItem {
     if (!(this.child instanceof TouchBar)) {
       this.child = new TouchBar(this.child)
     }
+    this.child.ordereredItems.forEach((item) => {
+      item._popover = item._popover || []
+      if (!item._popover.find(itemID => itemID === this.id)) item._popover.push(this.id)
```

Comment:
@kevinsawicki Will try this afternoon, but do `Set`'s get casted into `std::vector`'s automatically like normal arrays?